docs/ug/withGithubActions.md

```diff
@@ -21,19 +21,16 @@ You can use [_GitHub Actions_](https://github.com/features/actions) (together wi
 
 <include src="withTravis.md#section-fork-token" />
 
-{{ step(3)}} **Add the token/key as a secret:**
+{{ step(2)}} **Activate GitHub Actions on the forked repository**
 
-1. Go to the `Settings` page of your fork of the [publish-RepoSense](https://github.com/reposense/publish-RepoSense) repo.
-1. Click on the `Secrets` menu item on the left of that page.
-1. Click on `Add secret`.
-1. Add a new secret with the name `ACCESS_TOKEN` or `DEPLOY_KEY` (depending on your earlier choice) and the value of the token/key you copied earlier.<br>
-![GitHub Actions Secrets](../images/publishingguide-secrets.png "GitHub Actions Secrets")
+1. Go to the `Actions` page of your fork of the [publish-RepoSense](https://github.com/reposense/publish-RepoSense) repo.
+1. Click on the green button that says `I understand my workflows, go ahead and enable them` to enable GitHub Actions on your new repository.
```

Comment:
Can't seem to find this button on any of my repositories. Can show me where it comes and how?

Reply:
It needs to be forked to an account that has never forked the `publish-RepoSense` repository I think. I reproduced the above message by forking the repository to an empty organisational account.

Reply:
Is this just for `publish-RepoSense` first time? Or whenever someone who hasn't enabled GitHub Actions earlier needs to do this for the first time? In my opinion, I think it's the second case. If it is, then maybe better to add the fact that this step needs to be done only if enabling Actions for the first time. If it is not, then LGTM!

Reply:
I think it's the first case where you need to enable it for the first time you fork the repository. Also, GitHub only allows for forking a particular repository once per account, so it will refer to the first time that a particular account has forked the repository.